cell_classification/segmentation_data_prep.py

```diff
@@ -209,18 +210,44 @@ def check_input(self):
         else:
             self.selected_markers = self.selected_markers
 
+            # check if selected markers are in conversion matrix
```

Comment:
I think `data_folders` can get changed to `data_dir`. We should add a check that this is a valid directory. Then `data_folders` can be generated automatically with `ark.utils.io_utils.list_folders` by listing all of the folders within the directory. We'll have hundreds to thousands of folders in these datasets